tardis/energy_input/tests/test_gamma_ray_channel.py

```diff
@@ -17,8 +17,14 @@
     create_inventories_dict,
     calculate_total_decays,
     create_isotope_decay_df,
+    time_evolve_cumulative_decay,
 )
 
+from tardis.energy_input.gamma_ray_transport import get_taus
+from tardis.energy_input.util import KEV2ERG
```

Comment:
this is an astropy units thing

Reply:
astropy units have been used. Do you want separately in this file?